lib/api/CDataFrameAnalyzer.cc

```diff
@@ -35,6 +37,7 @@ const char FINISHED_DATA_CONTROL_MESSAGE_FIELD_VALUE{'$'};
 
 // Result types
 const std::string ROW_RESULTS{"row_results"};
+const std::string PROGRESS{"progress"};
```

Comment:
Since this is now inconsistent with `progress` from forecasting, maybe call it `progressPercent`.